backend/config.py.example

```diff
@@ -1,22 +1,20 @@
-# Rename this file to config.py and fill SQLALCHEMY_DATABASE_URI with your's data.
+ Rename this file to config.py and fill db_credentials with your's data.
+import os
+
 
 class Config(object):
+    # Change database credentials to yours.
+    db_credentials = 'postgres://user:password@localhost/db_name'
```

Comment:
If you want to use separate config variable for DB connection string, please make it naming style according to variables below.